### PR TITLE
card component completed

### PR DESCRIPTION
**Pull request by MayurChawla**
Hey I have added a component. Critical comments appreciated

!

### CODE REVIEW

File: Cards/Cardstyle.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+.card .card-container .card-text {
+    padding: 1rem;
+}
+.card .card-container .card-text .card-heading {
+    font-size : 1.25rem;
+    line-height: 2rem;
+}
+.card .card-container .card-text .card-subheading {
+    font-size: 0.875rem;
+    color: #0000008A;
```
Comment (Hamdrive) [inline Cards/Cardstyle.css]: preferred to use CSS variables for colors, define them in another stylesheet and use the variable name here using `var()`

File: Cards/Cardstyle.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    border: 0px;
+    background-color: transparent;
+    border-radius: 4px;
+    cursor: pointer;
+}
+
+.card-actions .card-buttons button {
+    height: 36px;
+    font-weight: 600;
+    letter-spacing: .0892857143em;
```
Comment (Hamdrive) [inline Cards/Cardstyle.css:80]: refrain from using `rem` for small measurements, instead use `px` upto 12px
Why such a specific decimal value for letter-spacing is used here?

File: Cards/Cardstyle.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    height: 12.25rem;
+}
+.custom-design-card .card-container .card-information {
+    padding-top: 0;
+}
+.custom-design-card .card-actions .card-buttons button:hover,
+.custom-design-card .card-actions .card-icons div i:hover {
+    background-color: var(--primary-color);
+    color: white;
+}
```
Comment (Hamdrive) [inline Cards/Cardstyle.css]: Add extra new line to remove this EOF

File: index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component Library</title>
+</head>
+<body>
+    This is a normal html page for the Component Library.
+</body>
+</html>
```
Comment (Hamdrive) [inline index.html:12]: Add extra new line to remove this EOF

**Review by Hamdrive — COMMENTED** [review]
Make the mentioned changes

File: index.html (hunk 4 above)
Comment (MayurChawla) [reply index.html:12]: noted

File: Cards/Cardstyle.css (hunk 2 above)
Comment (MayurChawla) [reply Cards/Cardstyle.css:80]: i actually took this value from component itself, that's why i kept the same

File: Cards/Cardstyle.css (hunk 1 above)
Comment (MayurChawla) [reply Cards/Cardstyle.css]: yes.. resolved

**Review by aurobindoGupta — APPROVED** [review]
nice work

**Pull request merged**